sdk/identity/Azure.Identity/README.md

```diff
@@ -276,16 +276,15 @@ client secret and certificate are both present, the client secret will be used.
 
 As of version 1.10.0, accessing resources protected by [Continuous Access Evaluation (CAE)][cae] is possible on a per-request basis. This behavior can be enabled by setting the `IsCaeEnabled` property of `TokenRequestContext` via its constructor. CAE isn't supported for developer and managed identity credentials.
 
-
 ## Token caching
 
-Token caching is a feature provided by the Azure Identity library that allows apps to:
+*Token caching* is a feature provided by versions 1.8.0 and later of the Azure Identity library. The feature allows apps to:
```

Comment:
Even though this is in the MI section, should we specify that this only refers to `ManagedIdentityCredential` here just to avoid confusion?